src/mri/exp_mult_mex.jl

```diff
@@ -0,0 +1,162 @@
+using Random
+#using MIRT: max_percent_diff
+#using MIRT: exp_xform
+
+export exp_mult_mex
+
+"""
+    D = exp_mult_mex(A, u, v)
+in:
+    A	[N L]	complex matrix
```

Comment:
Use - or * at start of each row for these arguments so they appear as a bulleted list.
and use backticks to enclose variables.
see all the examples in MIRT